docs/src/man/getting_started.md

```diff
@@ -45,7 +45,9 @@ julia> df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
 
 ```
 
-Columns can be directly (i.e. without copying) accessed via `df.col` or `df[!, :col]`. The latter syntax is more flexible as it allows passing a variable holding the name of the column, and not only a literal name. Note that column names are symbols (`:col` or `Symbol("col")`) rather than strings (`"col"`). Columns can also be accessed using an integer index specifying their position.
+Columns can be directly (i.e. without copying) accessed via `df.col` or `df[!, :col]`. The latter syntax is more flexible as it allows passing a variable holding the name of the column, and not only a literal name. Note that column names are symbols (`:col` or `Symbol("col")`) rather than strings (`"col"`). Columns can also be accessed using an integer index specifying their position. 
+
+Notice that, changing the elements of the column returned by `df[!, :col]` will affect the values stored in the original `df`, while `df[:, :col]` returns a copy of the column, and changing it does not change `df`.
```

Comment:
AFAICT `df[:, :col]` hasn't been mentioned yet at this point. Can you rephrase this so that it introduces `:`, rather than making it sound like users already know it?

Reply:
Fixed in https://github.com/JuliaData/DataFrames.jl/pull/2000/commits/c1de00de14e23f0fc19aa208fbbf5c5b130068eb